Packs/CortexAttackSurfaceManagement/Scripts/RemediationPathRuleEvaluation/RemediationPathRuleEvaluation.py

```diff
@@ -0,0 +1,225 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+"""
+Script for evaluating and finding a remediation path rule that matches for
+a given alert
+"""
+
+import traceback
+from typing import Any, Dict, List
+
+# should match IncidentSeverity in CommonServerPython
+SEVERITY_MAP = {
+    "unknown": 0,
+    "info": 0.5,
+    "low": 1,
+    "medium": 2,
+    "high": 3,
+    "critical": 4,
+}
+
+
+def match_remediation_rule(alert_context: Dict[str, Any], rules: List) -> List:
+    """
+    For the given alert (with the field in alert_context), finds the most recently created
+    remediation path rule where the alert meets the rule's criteria
+
+    Args:
+        alert_context: dictionary of relevant alert fields to check against
+        rules: list of remediation path rules to evaluate
+
+    Returns: the matching rule (list of 1)
+
+    """
+    matched_rules = []
+    # loop through remediation rules
+    if isinstance(rules, dict):
+        rules = [rules]
+    for rule in rules:
+        match = True
+        conditions = rule["criteria"]
+        if len(conditions) == 0:
+            demisto.info(
+                "Invalid Remediation Path Rule - no criteria condition to evaluate."
+            )
+            match = False
+        else:
+            # check criteria conjunction
+            if rule.get("criteria_conjunction") == "AND":
```

Comment:
Can this be changed to an `elif`?

Reply:
possibly, but I don't really see that making the code more simple/readable and not sure what the ending `else` would then be as well